.github/scripts/test_print_latest_commits.py

```diff
@@ -2,12 +2,28 @@
 from typing import Any, List, Dict
 from print_latest_commits import isGreen, WorkflowCheck
 
+workflowNames = [
+    "pull",
+    "trunk",
+    "Lint",
+    "linux-binary-libtorch-pre-cxx11",
+    "android-tests",
+    "windows-binary-wheel",
+    "periodic",
+    "docker-release-builds",
+    "nightly",
+    "pr-labels",
+    "Close stale pull requests",
+    "Update S3 HTML indices for download.pytorch.org",
+    "Create Release"
+]
+
 class TestChecks:
     def make_test_checks(self) -> List[Dict[str, Any]]:
```

Comment:
It might be easier/more representative to pull test data from prod and cache it like they do in test_trymerge.py.